product_pricelist_supplierinfo/tests/test_product_supplierinfo.py

```diff
@@ -38,7 +38,6 @@ def setUpClass(cls):
         })
         cls.pricelist = cls.env['product.pricelist'].create({
             'name': 'Supplierinfo Pricelist',
-            'discount_policy': 'without_discount',
```

Comment:
Why removing this?

Reply:
Because it is a field added by `sale` and `product_pricelist_supplierinfo` does not depend on `sale`

Reply:
OK, thanks for explaining. I suppose this doesn't have side effects.